test/models/work_test.rb

```diff
@@ -0,0 +1,91 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { Work.new(title: ' Talking Rain', publication_year:1990, creator: 'tangering') }
+
+  it "must be valid" do
+    value(work).must_be :valid?
+  end
+
+  it 'has required fields' do
+    fields = [:title, :publication_year, :creator]
+
+    fields.each do |field|
+      expect(work).must_respond_to field
+    end
+  end
+
+  describe 'Relationships' do
+    let(:work) { Work.new(title: ' Talking Rain', publication_year:1990, creator: 'tangering') }
+    it 'can have many votes' do
+
+      work.votes << Vote.first
+      votes = work.votes
+
+      expect(votes.length).must_be :>=, 1
+      votes.each do |vote|
+        expect(vote).must_be_instance_of Vote
+      end
+    end
+
+    it 'can only be voted on by a unique user once' do
+      # first_vote = Vote.first
```

Comment:
I would suggest here, creating a new vote which is identical to one from your fixtures.